t/26-video_stream.t

```diff
@@ -151,6 +155,12 @@ subtest 'frame parsing - ustreamer' => sub {
     # ustreamer requires pack("D") support, not availabe in openSUSE Leap 15.5's Perl
     eval { $_ = pack("D", 1.0); };
     plan skip_all => 'packing long double is not supported' if $@;
+    # frame unpacking is only correct on little-endian 64-bit arches, in
+    # practice ustreamer is only used on aarch64, restrict tests to
+    # aarch64 and x86_64 (x86_64 for CI convenience)
+    # see https://progress.opensuse.org/issues/161969
```

Comment:
Please also add something like `See: https://progress.opensuse.org/issues/161969` to your PR desc/commit message.

Reply:
@AdamWill This was the "requested change" I meant to ask for.